app/src/main/java/com/pluu/webtoon/db/EpisodeyUseCase.kt

```diff
@@ -0,0 +1,20 @@
+package com.pluu.webtoon.db
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/5475b2548342ec66d67b18c8c8d71379a80ef577/app/src/main/java/com/pluu/webtoon/db/EpisodeyUseCase.kt#L1"></span>class EpisodeUseCase should be declared in a file named EpisodeUseCase.kt (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>